Pear/Networking/Endpoints.swift

```diff
@@ -104,17 +67,7 @@ extension Endpoint {
     static func getUserInterests() -> Endpoint {
         Endpoint(path: "/user/interests/")
     }
-    
-    /// [GET] Get social media of the user.
-    static func getUserSocialMedia(netId: String) -> Endpoint {
-        Endpoint(path: "/user/socialMedia/", queryItems: [URLQueryItem(name: "netID", value: netId)], headers: standardHeaders)
-    }
-    
-    /// [GET] Get goals of the user.
-    static func getUserGoals(netId: String) -> Endpoint {
-        Endpoint(path: "/user/goals/", queryItems: [URLQueryItem(name: "netID", value: netId)], headers: standardHeaders)
-    }
-    
+
     /// [GET] Get availabilities of user by netID.
     static func getUserAvailabilities(netId: String) -> Endpoint {
```

Comment:
if we've fully committed to removing availabilities then this is something that's going to go away too